app/models/order.js

```diff
@@ -25,6 +25,7 @@ export default Model.extend({
   peopleHelped: attr('number'),
   goodcityRequests:   hasMany('goodcity_request', { async: false }),
   district: belongsTo('district', {async: false}),
+  bookingType:   belongsTo('booking_type', { async: false }),
```

Comment:
if we're adding this to the order model, I assume we need to remove it from the order_transport model. I think that's missing :)

Reply:
Yeah @patrixr .. removed it.. forgot to commit that file.. Thanks 😅

Reply:
Cool :D

Reply:
Committed change 👍